environments/prod/versions.yaml

```diff
@@ -1,20 +1,20 @@
 versions:
-  hocs_audit: 2.17.3
-  hocs_case_creator: 1.10.2
+  hocs_audit: 2.18.3
+  hocs_case_creator: 1.12.8
   hocs_case_migrator:
-  hocs_casework: 3.11.2
-  hocs_docs: 3.5.1
-  hocs_docs_converter: 1.6.2
-  hocs_extracts: 2.17.3
-  hocs_frontend: 2.22.0
-  hocs_info_service: 3.2.24
-  hocs_management_ui: 2.1.1
+  hocs_casework: 3.16.0
+  hocs_docs: 3.6.6
```

Comment:
Are we taking the latest patch regardless, the docs say this should be `3.6.4`. I'm not fussed either way...

Reply:
They're latest patches that have passed regression. Don't hate the player etc

Reply:
So they're from the QA versions file except some of our tools